orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -130,11 +132,26 @@ OrtValue create_ort_value(
   }
 
   OrtValue ort_tensor;
-  CreateMLValue(
-    tensor.data_ptr(),
-    ort_scalar_type_from_aten(tensor.scalar_type()),
-    tensor.sizes().vec(),
-    &ort_tensor);
+  if (tensor.device().type() == at::kORT) {
```

Comment:
if input is a tensor on ORT device, it should meet the condition at line 129. why you need this?

Reply:
not sure I fully understand but what you mean is if tensor is on ORT device, then these lines will be triggered, right?

```
 auto* impl = dynamic_cast<ORTTensorImpl*>(tensor.unsafeGetTensorImpl());
  if (impl) {
    return impl->tensor();
  }
```